torch/csrc/distributed/autograd/context/context.cpp

```diff
@@ -115,7 +115,7 @@ std::shared_ptr<torch::autograd::GraphTask> DistAutogradContext::
 }
 
 void DistAutogradContext::setGraphTask(
-    std::shared_ptr<torch::autograd::GraphTask> graphTask) {
+    const std::shared_ptr<torch::autograd::GraphTask>& graphTask) {
```

Comment:
Now the std::move in the function is useless though? Why was this changed?

Reply:
Reverted